knexfile.ts

```diff
@@ -1,33 +1,43 @@
 import type { Knex } from 'knex';
+import dotenv from 'dotenv';
+
+dotenv.config(); 
```

Comment:
Dica: também funcionaria :wink: :
```suggestion
import 'dotenv/config';
```